VoodooI2CHID/VoodooI2CTouchscreenHIDEventDriver.cpp

```diff
@@ -167,16 +198,22 @@ void VoodooI2CTouchscreenHIDEventDriver::fingerLift() {
     start_scroll = true;
     uint64_t now_abs;
     clock_get_uptime(&now_abs);
-    
-    dispatchDigitizerEventWithTiltOrientation(now_abs, last_id, kDigitiserTransducerFinger, 0x1, 0x0, last_x, last_y);
-    
+
+    dispatchDigitizerEventWithTiltOrientation(now_abs, last_id, kDigitiserTransducerFinger, 0x1, HOVER, last_x, last_y);
+#ifdef VOODOO_I2C_TOUCHSCREEN_DEBUG
```

Comment:
Can we please remove the `VOODOO_I2C_TOUCHSCREEN_DEBUG` preprocessor variable and remove the log as it's device specific?